src/coprocessor/codec/batch/lazy_column_vec.rs

```diff
@@ -227,24 +201,47 @@ mod tests {
 
     use crate::coprocessor::codec::datum::{Datum, DatumEncoder};
 
-    /// Helper method to generate raw row ([u8] vector) from datum vector.
-    fn raw_row_from_datums(datums: impl AsRef<[Option<Datum>]>, comparable: bool) -> Vec<Vec<u8>> {
-        datums
+    /// Pushes a raw row. There must be no empty datum.
```

Comment:
Do we need to use `///` for these two functions?

Reply:
Not required But still nice to have some comment for that :)

Reply:
How about just using `//`?

Reply:
Why should we do that? 🤔

Reply:
It's a part of the tests, so I'm not sure if rustdocs should contain this kind of comments.

Reply:
I think it's just fine for test purpose functions to have rustdocs describing their functionality, so that IDE users can know the description of the function immediately. There is nothing special for "mod tests".